src/main/resources/db/changelog/db.changelog.xml

```diff
@@ -514,6 +514,21 @@
             </column>
         </addColumn>
     </changeSet>
+  
+    <changeSet id="040-create-tweet-table" author="Michał.Chmielarz">
+
+        <createSequence sequenceName="TWEET_SEQ"/>
+
+        <createTable tableName="TWEET">
+            <column name="ID" type="bigint">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="CONTENT" type="varchar(140)">
```

Comment:
As I said, I am not sure whether longer messages are always rejected as many long links will be shortened using `t.co` service.